File: VPC.WebApi/Model/Entities.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace VPC.WebApi.Model
{
    public class Entities
    {
        public string DisplayName { get; set; }
        public string Name { get; set; }
        public string PluralName { get; set; }
        public bool SupportWorkflow { get; set; }
        public string EntityType { get; set; }
        public Configuration Configurations { get; set; }
        public List<string> Subtypes { get; set; }
        public List<FieldModel> Fields { get; set; }
        public List<Relation> Relations { get; set; }
        public List<Operation> Operations { get; set; }
        public List<Tasks> Tasks { get; set; }
        public List<Entities> RelatedEntities { get; set; }
        public List<RowLevelOperations> RowLevelOperations { get; set; }
    }
}

File: VPC.Client/ClientApp/src/app/model/entities.ts
import { Configuration } from './configuration';
import { FieldModel } from './fieldmodel';
//import { ITreeNode} from '../model/treeNode';
import { Relation } from './relation';
import { Operation } from './operation';
import { Tasks } from './tasks';
import { RowLevelOperations } from './rowleveloperations';
import { Rule } from './rule';

export class Entities {
  displayName: string;
  name: string;
  pluralName: string;
  supportWorkflow: boolean;
  entityType: string;
  configurations: Configuration;
  subtypes: string[];
  fields: FieldModel[];
  relations: Relation[];
  operations: Operation[];
  tasks: Tasks[];
  rules: Rule[];
  relatedEntities: Entities[];
  rowLevelOperations: RowLevelOperations[];
  detailEntities:any;
  activityEntity:Entities;
}
